README.md

```diff
@@ -195,6 +195,7 @@ A Visa Token Service (VTS) response like this: <br />
 	"cardType": "string",	
 	"cardIssuedInCountryCode": "string",
 	"maskedCardNumber":"string",
+        "cardBin":"string",
```

Comment:
```suggestion
	"cardBin":"string",
```

Tabs vs. spaces 🤷‍♂️

Reply:
@ct-clearhaus: You're correct. In effect, we'll be returning 8+4 digits (when combining maskedCardNumber + cardBin). This has been assessed internally to comply with PCI-DSS

Reply:
@mniss, thanks a bunch! Maybe I'm slow, but can you ELI5 me how this conforms to PCI DSS Req. 3.3:

<kbd>
<img width="535" alt="Screenshot 2022-05-03 at 09 22 57" src="https://user-images.githubusercontent.com/6168505/166417250-5a026aea-44f3-4383-8166-93dca6ab8eba.png">
</kbd>

Maybe you're expecting the recipient to get the response in a CDE?

Reply:
@ct-clearhaus , I've sent you an e-mail :)